Fitbit.Portable/JsonDotNetSerializerExtensions.cs

```diff
@@ -83,21 +81,18 @@ internal static List<UserProfile> GetFriends(this JsonDotNetSerializer serialize
             return friends.Children().Select(serializer.Deserialize<UserProfile>).ToList();           
         }
 
-
-
-
-
         /// <summary>
         /// GetTimeSeriesDataList has to do some custom manipulation with the returned representation
         /// </summary>
         /// <param name="serializer"></param>
-        /// <param name="timeSeriesDataJson"></param>
+        /// <param name="date"></param>
+        /// <param name="heartRateIntradayJson"></param>
         /// <returns></returns>
         internal static HeartActivitiesIntraday GetHeartRateIntraday(this JsonDotNetSerializer serializer, DateTime date, string heartRateIntradayJson)
         {
             if (string.IsNullOrWhiteSpace(heartRateIntradayJson))
             {
-                throw new ArgumentNullException("heartRateIntradayJson", "heartRateIntradayJson can not be empty, null or whitespace.");
+                throw new ArgumentNullException(nameof(heartRateIntradayJson), "heartRateIntradayJson can not be empty, null or whitespace.");
```

Comment:
Should you add nameof() in the message as well? Or remove the name of the arg there?